src/internal/translations/en-US.json

```diff
@@ -584,6 +584,56 @@
         "comment": "",
         "limit": 0
     },
+    "properties.points.title": {
+        "value": "Points",
+        "comment": "",
+        "limit": 0
+    },
+    "properties.points.groupNearbyPoints": {
+        "value": "Group nearby points",
+        "comment": "",
+        "limit": 0
+    },
+    "properties.points.size.title": {
+        "value": "Point Size",
+        "comment": "",
+        "limit": 0
+    },
+    "properties.points.size.extra_small": {
+        "value": "0.5x",
+        "comment": "",
```

Comment:
Please provide context. Sth like "Multiplicator of size of a point on map. Means 50% of size."
Applies to all multiplicators below.